extension/js/common/api/account-servers/flowcrypt-com-api.ts

```diff
@@ -34,36 +32,34 @@ export namespace BackendRes {
 
 export class FlowCryptComApi extends Api {
 
-  public static loginWithOpenid = async (acctEmail: string, uuid: string, idToken: string): Promise<void> => {
+  private static getAuthorizationHeader = (idToken: string) => {
+    return { Authorization: `Bearer ${idToken}` };
+  };
+
+  public static loginWithOpenid = async (idToken: string): Promise<void> => {
```

Comment:
I believe `loginWithOpenid` can be removed as well. The original purpose was to register a new `uuid` and register potentially new account.

uuids are no longer used and new accounts are automatically registered when calling another API with id token. Therefore, I expect it should work without `loginWithOpenid`.